drivers/cc110x/include/cc110x_constants.h

```diff
@@ -516,6 +537,39 @@ extern "C" {
  */
 #define CC110X_FIFO_SIZE                64
 
+/**
+ * @brief   Value of the bits 7-2 of the PKTCTRL1 configuration register used
+ *          in this driver
+ */
+#define CC110X_PKTCTRL1_VALUE           0x00
+
+/**
+ * @name    Possible address matching policies
+ *
+ * See page 73 in the data sheet. The policy should be combined with
+ * @ref CC110X_PKTCTRL1_VALUE via bitwise or. (Only modes compatible with the
+ * driver are defined.)
+ *
+ * @{
+ */
+/**
+ * @brief   Accept incoming frames regardless of address
+ */
+#define CC110X_PKTCTRL1_ADDR_ALL        0x00
+/**
+ * @brief   Accept frames with with matching address broadcast address
```

Comment:
Did you want to write: `Accept frames with matching address or broadcast address`?

Reply:
Yep, thanks! Fixed :-)

Reply:
I don´t want to be pedantic but there still is the double "with" :-)

Reply:
Done. Thanks for double checking!